src/logos/codeSquareLogo.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+export const CodeSquareLogo = () => {
```

Comment:
I wonder if we should move all of these logos into the CDN similar to how we already did for the top you.com logo

Reply:
yeah, if possible this is a good idea, since we may want to udpate and we'll be reliant on users to pull them elsewise.